mysql/datadog_checks/mysql/config.py

```diff
@@ -34,8 +34,8 @@ def __init__(self, instance):
         self.full_statement_text_samples_per_hour_per_query = instance.get(
             'full_statement_text_samples_per_hour_per_query', 1
         )
-        self.statement_samples_config = instance.get('statement_samples', {}) or {}
-        self.statement_metrics_config = instance.get('statement_metrics', {}) or {}
+        self.statement_samples_config = instance.get('query_samples', {}) or {}
+        self.statement_metrics_config = instance.get('query_metrics', {}) or {}
```

Comment:
We might want to support the legacy behavior.

Reply:
Good point, updated